src/actions/mesInformations.actions.js

```diff
@@ -0,0 +1,130 @@
+import Joi from 'joi';
+import { userService } from '../services/user.service';
+import { mesInformationsService } from '../services/mesInformations.service';
+
+export const mesInformationsActions = {
+  verifyFormulaire,
+  getContratActif,
+  updateField,
+  updateContratActif,
+  getInformationsManquantes,
+  getPlusTard,
+};
+
+function verifyFormulaire(form) {
+  let errors = [];
+  // eslint-disable-next-line max-len
+  let { typeContrat, dateDebut, dateFin } = form;
+
+  errors.push({
+    typeContrat: (Joi.object({
+      typeContrat: Joi.string().trim().required()
```

Comment:
pas de valeurs précises en in ?

Reply:
C'est un champs radio

Reply:
yes pour le CRA on avait brider avec un .valid()